src/libs/fsdrv/lv_fs_littlefs.c

```diff
@@ -0,0 +1,190 @@
+#include "../../../lvgl.h"
+#if LV_USE_FS_LITTLEFS
+
+#include <lfs.h>
```

Comment:
I wonder where this `lfs.h` comes from.

To help other developers how to install and use LittleFS, could you please add new document to the 3rd party libraries folder? 
https://github.com/lvgl/lvgl/tree/master/docs/libs

It is enough to write down how to install / clone LittleFS, like in `libpng` doc:
https://docs.lvgl.io/master/libs/libpng.html

You can also mention in File systems doc that LittleFS is now supported, too: https://github.com/lvgl/lvgl/blob/master/docs/libs/fs.rst

Reply:
I'm not a good English writer (nor a french writer as well :( ), but if you think something is mandatory and missing inside the documentation, tell me :pray: